internal/impl/kube.go

```diff
@@ -323,12 +323,12 @@ func (r *replicaSetInfo) buildContainer() (corev1.Container, error) {
 	// docker image.
 	r.dep.App.Binary = fmt.Sprintf("/weaver/%s", filepath.Base(r.dep.App.Binary))
 	kubeCfgStr, err := proto.ToEnv(&ReplicaSetConfig{
-		Namespace:            r.namespace,
-		Deployment:           r.dep,
-		ReplicaSet:           r.name,
-		ComponentsToStart:    r.components,
-		InternalPort:         int32(r.internalPort),
-		TraceExporterService: r.traceServiceURL,
+		Namespace:         r.namespace,
+		Deployment:        r.dep,
+		ReplicaSet:        r.name,
+		ComponentsToStart: r.components,
+		InternalPort:      int32(r.internalPort),
+		TraceServiceUrl:   r.traceServiceURL,
```

Comment:
nit: s/Url/URL/

Reply:
TraceServiceUrl is the autogenerated proto field `trace_service_url`. Unless we define it as trace_serviceURL, I think protoc will convert URL to Url.